Sources/Workspace/Workspace.swift

```diff
@@ -1619,24 +1619,27 @@ extension Workspace {
             group.enter()
 
             guard case .remote(let url, let checksum, _) = artifact.source, let destination = path(for: artifact) else {
+                group.leave()
```

Comment:
good catch. how about a `defer { group.leave() }` next to `group.enter()` then a second enter/leave pair for downloader.downloadFile?

Reply:
Either way works for me 👍